### PR TITLE
Separate UPPAAL settings and its serialization from View.

### PR DESCRIPTION
**Pull request by benedekh**

### CODE REVIEW

File: plugins/core/hu.bme.mit.gamma.uppaal.verification/src/hu/bme/mit/gamma/uppaal/verification/settings/UppaalSettings.xtend [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+	static class Builder {
+		UppaalSettings instance
+
+		new() {
+			this.instance = new UppaalSettings
+		}
+
+		def Builder searchOrder(String value) {
+			instance.searchOrder = value
+			this
```
Comment (grbeni) [inline plugins/core/hu.bme.mit.gamma.uppaal.verification/src/hu/bme/mit/gamma/uppaal/verification/settings/UppaalSettings.xtend:49]: In this project, we write the `return` keyword explicitly before the value to be returned in method bodies.

File: plugins/core/hu.bme.mit.gamma.querygenerator.gui/src/hu/bme/mit/gamma/querygenerator/application/View.java [hunk 2]
```diff
@@ -10,6 +10,30 @@
...
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.STATE_SPACE_REDUCTION_DEFAULT;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.STATE_SPACE_REDUCTION_NONE;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.STATE_SPACE_REPRESENTATION_DBM;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.STATE_SPACE_REPRESENTATION_DEFAULT;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.STATE_SPACE_REPRESENTATION_OA;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.STATE_SPACE_REPRESENTATION_UA;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.TRACE_DEFAULT;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.TRACE_FASTEST;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.TRACE_SHORTEST;
+import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.TRACE_SOME;
```
Comment (grbeni) [inline plugins/core/hu.bme.mit.gamma.querygenerator.gui/src/hu/bme/mit/gamma/querygenerator/application/View.java:35]: If every setting from UppaalSettings is used here, maybe we could consider using `import static hu.bme.mit.gamma.uppaal.verification.settings.UppaalSettings.*;` instead.

Comment (benedekh) [reply plugins/core/hu.bme.mit.gamma.querygenerator.gui/src/hu/bme/mit/gamma/querygenerator/application/View.java:35]: Eclipse replaced it automatically.

File: plugins/core/hu.bme.mit.gamma.uppaal.verification/src/hu/bme/mit/gamma/uppaal/verification/settings/UppaalSettings.xtend (hunk 1 above)
Comment (benedekh) [reply plugins/core/hu.bme.mit.gamma.uppaal.verification/src/hu/bme/mit/gamma/uppaal/verification/settings/UppaalSettings.xtend]: Fixed at every place.

**Review by grbeni — APPROVED** [review]
Approved as the minor requests have been addressed.

**Pull request merged**